Using_LIO_maps.Rmd

```diff
@@ -6,6 +6,11 @@ output:
   html_document:
     df_print: paged
 ---
+```{r, setup, include=FALSE}
+knitr::opts_chunk$set(message = FALSE)
+knitr::opts_chunk$set(warning = FALSE)
```

Comment:
Hey Jeremy, you can use 

`knitr::opts_chunk$set(message = FALSE, warning  = FALSE)` here to save space.